build/Targets/VSL.Imports.targets

```diff
@@ -59,15 +59,27 @@
       <RemoveAssemblyFoldersIfNoTargetFramework>false</RemoveAssemblyFoldersIfNoTargetFramework>
   </PropertyGroup>
 
+  <!-- Work around issue in dotnet/roslyn issue #5213 -->
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == 'DNXCore'">
+    <_TargetFrameworkDirectories>$([System.String]::Join(";", $([Microsoft.Build.Utilities.ToolLocationHelper]::GetPathToReferenceAssemblies(".NETPortable", "v5.0", ""))))</_TargetFrameworkDirectories>
+    <_FullFrameworkReferenceAssemblyPaths>$(_TargetFrameworkDirectories)</_FullFrameworkReferenceAssemblyPaths>
```

Comment:
The warning is produced by the task here: https://github.com/Microsoft/msbuild/blob/master/src/XMakeTasks/Microsoft.Common.CurrentVersion.targets#L1097

By setting these properties, this task call is skipped and the warning is not produced.

As far as I can tell, this shouldn't have any adverse effects. If at any time ResolveAssemblyReferences decided to look into the given directory, that's just the .NETPortable,v5.0 reference assembly directory, so it doesn't contain any reference assemblies anyway (i.e., it contains the same set of reference assemblies that DNXCore50 has -- none).